src/ansys/dpf/composites/_indexer.py

```diff
@@ -80,56 +114,31 @@ class PropertyFieldIndexerNoDataPointer:
     """Indexer for a property field with no data pointer."""
 
     def __init__(self, field: PropertyField):
```

Comment:
The implements are ordered now. All PropertyFieldIndexerfirst, then the FieldIndexer classes.